src/video/camerasource.cpp

```diff
@@ -337,7 +337,7 @@ void CameraSource::stream()
         }
 
       // Free the packet that was allocated by av_read_frame
-      av_free_packet(&packet);
+      av_packet_unref(&packet);
```

Comment:
I found stable ffmpeg v2.8 on Debian 8 [here](https://tracker.debian.org/pkg/ffmpeg)
In ffmpeg docs av_packet_unref exist in [v2.4](https://www.ffmpeg.org/doxygen/2.4/group__lavc__packet.html)

If something is wrong, please, tell me